package.json

```diff
@@ -16,34 +16,48 @@
     "cross",
     "platform"
   ],
+  "engines": {
+    "node": ">=18 <20"
+  },
+  "files": [
+    "index.js",
+    "src",
+    "docs",
+    "example",
+    "prebuilds",
+    "binding.gyp",
+    "LICENSE",
+    "README.md",
+    "requirements.txt"
+  ],
   "directories": {
     "test": "tests"
   },
   "scripts": {
     "configure": "node-gyp configure",
-    "build": "node-gyp build",
+    "build": "node-gyp-build && npm run prebuildify",
+    "prebuildify": "prebuildify -t 18.18.2 -t 20.5.1 --strip",
```

Comment:
What was the reason for moving away from `--all`? Since the specific targets are likely to rapidly become redundant for actual usage